### ISSUE TITLE
Error when installing with foxy

### ISSUE THREAD

**Issue by schmunk42** [sonatra/sonatra-form-extensions-bundle#2]
I ran into some errors when installing this package, tried in combination with other packages and standalone.

```
Merging Composer dependencies in the asset package
Updating yarn dependencies
Executing command (CWD): yarn upgrade --non-interactive
yarn upgrade v1.0.2
warning package.json: No license field
warning No license field
error Outdated lockfile. Please run `yarn install` and try again.
info Visit https://yarnpkg.com/en/docs/cli/upgrade for documentation about this command.
Fallback to previous state for the Asset package
Fallback to previous state for Composer
Reading ./composer.lock
Reading ./composer.lock

Installation failed, reverting ./composer.json to its original content.

                                                                  
  [Symfony\Component\Console\Exception\InvalidArgumentException]  
  The "no-dev" option does not exist.                             
                                                                  

Exception trace:
 () at phar:///opt/bin/composer/vendor/symfony/console/Input/Input.php:157
 Symfony\Component\Console\Input\Input->getOption() at /app/pkg/vendor/foxy/foxy/Fallback/ComposerFallback.php:171
 Foxy\Fallback\ComposerFallback->restorePreviousLockFile() at /app/pkg/vendor/foxy/foxy/Fallback/ComposerFallback.php:127
 Foxy\Fallback\ComposerFallback->restore() at /app/pkg/vendor/foxy/foxy/Solver/Solver.php:99
 Foxy\Solver\Solver->solve() at phar:///opt/bin/composer/src/Composer/Plugin/PluginManager.php(195) : eval()'d code:124
 Foxy\Foxy_composer_tmp0->solveAssets() at n/a:n/a
 call_user_func() at phar:///opt/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:171
 Composer\EventDispatcher\EventDispatcher->doDispatch() at phar:///opt/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:96
 Composer\EventDispatcher\EventDispatcher->dispatchScript() at phar:///opt/bin/composer/src/Composer/Installer.php:307
 Composer\Installer->run() at phar:///opt/bin/composer/src/Composer/Command/RequireCommand.php:180
 Composer\Command\RequireCommand->execute() at phar:///opt/bin/composer/vendor/symfony/console/Command/Command.php:266
 Symfony\Component\Console\Command\Command->run() at phar:///opt/bin/composer/vendor/symfony/console/Application.php:861
 Symfony\Component\Console\Application->doRunCommand() at phar:///opt/bin/composer/vendor/symfony/console/Application.php:208
 Symfony\Component\Console\Application->doRun() at phar:///opt/bin/composer/src/Composer/Console/Application.php:245
 Composer\Console\Application->doRun() at phar:///opt/bin/composer/vendor/symfony/console/Application.php:127
 Symfony\Component\Console\Application->run() at phar:///opt/bin/composer/src/Composer/Console/Application.php:100
 Composer\Console\Application->run() at phar:///opt/bin/composer/bin/composer:54
 require() at /opt/bin/composer:24

require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-suggest] [--no-update] [--no-scripts] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [--sort-packages] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--] [<packages>]...
```

A manual install works, but the above error occurs again.

```
/app/pkg # yarn upgrade --non-interactive
yarn upgrade v1.0.2
warning package.json: No license field
warning No license field
warning No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Rebuilding all packages...
success Saved lockfile.
success Saved 1 new dependency.
└─ @composer-asset/almasaeed2010--adminlte@2.4.0
warning No license field
Done in 3.80s.
```

----

While trying with a clean setup I got

```
Executing command (CWD): yarn install --non-interactive
yarn install v1.0.2
warning package.json: No license field
info No lockfile found.
warning No license field
[1/4] Resolving packages...
warning jvectormap@1.2.2: jvectormap is not maintened since Aug 2015. You can use jvectormap-next or jqvmap instead.
error Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads ssh://git@github.com/sonatra/sonatra-jquery-datetime-picker.git
Directory: /app/pkg
Output:
Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Maybe you don't see this error because you have a correct key for this URL?

```
/app/pkg # git ls-remote --tags --heads ssh://git@github.com/sonatra/sonatra-jquery-datetime-picker.git
The authenticity of host 'github.com (192.30.253.112)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no)? y
Please type 'yes' or 'no': yes
Warning: Permanently added 'github.com,192.30.253.112' (RSA) to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by francoispluchino** [comment]
The repository https://github.com/sonatra/sonatra-jquery-datetime-picker is public. But a SSH key without write access not work. I will use the short format in package dependencies `<username>/<repository>#<branch>`.

**Comment by francoispluchino** [comment]
Fixed by sonatra/sonatra-form-extensions@e82e3a20807b0916963ea6bb916de7cd0e0ea6a0.
